Library/Homebrew/extend/os/mac/hardware/cpu.rb

```diff
@@ -1,4 +1,4 @@
-require "os/mac/pathname"
+require "extend/os/mac/extend/pathname"
```

Comment:
This line doesn't look like it should go here. I'm guessing that it preexisted `extend/os/mac`. Can you try removing this line, and adding it to `extend/os/pathname.rb`?

Reply:
I'm not sure I understand...
I can add `require "extend/os/mac/extend/pathname" if OS.mac?` to `extend/os/pathname.rb?` but do you want this line removed from `cpu.rb`?

Reply:
got it

Reply:
> do you want this line removed from cpu.rb?